### PR TITLE
Jest and Storyshots for story snapshots

### LINKED ISSUE #4: Add tests
Add unit tests.

Since we already have Storybook stories, I think something like https://github.com/storybooks/storyshots could be worth exploring. To begin with our tests would simply be snapshots of the stories.

What do you think, @kennetvu?

### PR DESCRIPTION
**Pull request by alexanbj**
Added [Jest](https://github.com/facebook/jest) and [Storyshots](https://github.com/storybooks/storyshots) for automatic snapshot tests of the Storybook stories. This fixes #4.

Also, the Bulma CSS is no longer imported in index.js. Not everyone uses webpack, and some might want to roll their own Bulma variant with overriden variables.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
 # Rulma
 
 React components for [Bulma](https://github.com/jgthms/bulma)
+
+## Getting started
+#### Install
+```bash
+npm install --save rulma
+```
```
Comment (kennetvu) [inline README.md:9]: Should we add info for yarn? :D

File: README.md [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+
+#### Include Bulma CSS
```
Comment (kennetvu) [inline README.md:11]: We need to say something about how to install bulma through npm/yarn

File: stories/storyshots.test.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+import initStoryshots from 'storyshots';
+
+// Turn the Storybook stories into Jest snapshots
+initStoryshots();
```
Comment (kennetvu) [inline stories/storyshots.test.js:4]: snapshot files are generated ?

Comment (alexanbj) [reply stories/storyshots.test.js:4]: I'm not sure I understand the question? 

[Storyshots](https://github.com/storybooks/storyshots) generates the Jest snapshots automatically for us. So we don't have to create snapshots tests for each and every component ourselves.

File: README.md (hunk 1 above)
Comment (alexanbj) [reply README.md:9]: I don't know. Feels unnecessary? I think Yarn users should be able to cope quite well even with current installation instructions?

File: README.md (hunk 2 above)
Comment (alexanbj) [reply README.md:11]: We already have Bulma as a dependency, so it is actually available for the user. But maybe we should move it to a dev dependency instead?

File: stories/storyshots.test.js (hunk 3 above)
Comment (kennetvu) [reply stories/storyshots.test.js:4]: nvm :)

**Pull request merged**